### PR TITLE
Update DSEthiop23.xml

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Some changes, harmonizing.

### CODE REVIEW

File: DayrSuryan/DSEthiop23.xml [hunk 1]
```diff
@@ -62,99 +62,102 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                     ቀዳማይ፡ ክፍሊ፡ መእታዊ፡ (መቅድማት)
+                     ዓንቀጽ፡ (፩)
+                     ናይ፡ እስክንድርያ፡ ኦርቶዶክስ፡ ናይ፡ ግብጺ፡ ቤተ፡ ክርስቲያን፡ ሓዋርያዊት፡ ቤተ፡ ክርስቲያን፡ እያ፡ ርክባ፡ ከኣ፡ ምስተን፡ ናይ፡ ምብራቅ፡
+                     (oriental orthodox) ኦርቶዶክስ፡  ብእምነትን፡ <pb n="2r"/>ምሥጢራተ፡ ቤተ፡ ክርስቲያንን፡ ልማድን፡ (ecclesiastical tradition) ምሳኣ፡
+                     ርክብ (ኅብረት፡ ዘለወን፡ እይ
                      </incipit>
                      <explicit xml:lang="ti">
                         <locus target="#24r"/>
-                        መ_እቲ፡  ዝልወጥ፡  ዓንቀጽ፡  ወይክፍሊ፡  ቅ/ፓፓ፡  ብዘይብሉ፡  ጊዜ፡  ክኸውን፡  ኣይከኣልን፡  እዩ። ተርጉዋሚኡን፡  ጻሓፊኡን፡  መቃርዮስ፡  አቡን፡  ዲበኵሉ፡  ፲፱የኲሩ 1994
+                        መ_እቲ፡  ዝልወጥ፡  ዓንቀጽ፡ ወይክፍሊ፡ ቅ/ፓፓ፡ ብዘይብሉ፡ ጊዜ፡ ክኸውን፡
```
Comment (thea-m) [inline DayrSuryan/DSEthiop23.xml:155]: Is _ in the manuscript? I don't find the page in the images.

File: DayrSuryan/DSEthiop23.xml [hunk 2]
```diff
@@ -62,99 +62,102 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
-                     ናይ፡  እስክንድርያ፡  ኦርቶዶክስ፡  ናይ፡  ግብጺ፡  ቤተ፡  ክርስቲያን፡  ሓዋርያዊት፡  ቤተ፡  ክርስቲያን፡  እያ፡  ርክባ፡  ከኣ፡  ምስተን፡  ናይ፡  ምብራቅ
-                     ፡  (oriental orthodox) ኦርቶዶክስ፡   ብእምነትን፡  <pb n="2r"/>ምሥጢራተ፡  ቤተ፡  ክርስቲያንን፡  ልማድን፡  (ecclesiastical tradition) ምሳኣ
-                     ፡  ርክብ (ኅብረት፡  ዘለወን፡  እይ     
+                         በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ አሐዱ፡ አምላክ፡ አሜን ።
+                         ዓንዲ፡ ሕጊ፡ ናይ፡ ግብጺ፡ ኦርቶዶክስ፡ ቤተ፡ ክርስቲያን፡ ሲኖዶስ። ንናይ፡ ግብጺ፡ እስክንድርያ፡ ኦርቶዶክስ፡ ቤተ፡ ክርስቲያንን፡ ናይ፡
+                         ሓውርያ፡ ማርቆስ፡ ሃገረ፡ ስብከትን፡ ኵሉ፡ መማኀዳደሪ።...
+                     ቀዳማይ፡ ክፍሊ፡ መእታዊ፡ (መቅድማት)
+                     ዓንቀጽ፡ (፩)
+                     ናይ፡ እስክንድርያ፡ ኦርቶዶክስ፡ ናይ፡ ግብጺ፡ ቤተ፡ ክርስቲያን፡ ሓዋርያዊት፡ ቤተ፡ ክርስቲያን፡ እያ፡ ርክባ፡ ከኣ፡ ምስተን፡ ናይ፡ ምብራቅ፡
+                     (oriental orthodox) ኦርቶዶክስ፡  ብእምነትን፡ <pb n="2r"/>ምሥጢራተ፡ ቤተ፡ ክርስቲያንን፡ ልማድን፡ (ecclesiastical tradition) ምሳኣ፡
```
Comment (thea-m) [inline DayrSuryan/DSEthiop23.xml:150]: Do you remember if the English in paratheses is written in the manuscript? I don't find the right image, but I think you have it?

File: DayrSuryan/DSEthiop23.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop23.xml:155]: I think it can be dropped.

File: DayrSuryan/DSEthiop23.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop23.xml:150]: I have no images, but I cannot imagine that anyone of us could add this.

Comment (thea-m) [reply DayrSuryan/DSEthiop23.xml:150]: No, not really...

**Pull request merged**